changelog/17678.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Remove debug symbols from release binaries.
```

Comment:
From the writeup, we expect the primary externally-visible change to be "reduced binary size", and I think that might be a better way of describing the change in the changelog.

Reply:
:+1: